sitemap.xml

```diff
@@ -0,0 +1,184 @@
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=es</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=et</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=fi</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=fr</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=hr</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=id</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=it</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=ja</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=ko</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=nl</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=pl</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=pt-br</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=ru</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=sk</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=sv</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=vi</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=zh-cn</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=zh-hk</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=zh-tw</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://api.electricitymap.org/open-source/</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://www.tmrow.com/blog/</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
```

Comment:
Let's remove all `api.electricitymap.org` and `tmrow.com` entries, as they live on different domains/subdomains :)